packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -1034,6 +1038,7 @@ class _DropdownMenuState<T> extends State<DropdownMenu<T>> {
                     : null,
             suffixIcon: trailingButton,
           ).applyDefaults(effectiveInputDecorationTheme),
+          textInputAction: widget.textInputAction,
```

Comment:
Super nit: we can move this line above `style:...`. Just want to move the one-line settings together:)

Reply:
Done